services/contract_delete.proto

```diff
@@ -19,50 +28,75 @@ package proto;
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- * ‍
  */
+
 option java_package = "com.hederahashgraph.api.proto.java";
 // <<<pbj.java_package = "com.hedera.hapi.node.contract">>> This comment is special code for setting PBJ Compiler java package
 option java_multiple_files = true;
 
 import "basic_types.proto";
 
 /**
- * At consensus, marks a contract as deleted and transfers its remaining hBars, if any, to a
- * designated receiver. After a contract is deleted, it can no longer be called.
- * 
- * If the target contract is immutable (that is, was created without an admin key), then this
- * transaction resolves to MODIFYING_IMMUTABLE_CONTRACT.
- * 
- * --- Signing Requirements ---
- * 1. The admin key of the target contract must sign.
- * 2. If the transfer account or contract has receiverSigRequired, its associated key must also sign
+ * Delete a smart contract, and transfer any remaining HBAR balance to a
+ * designated account.
+ *
+ * If this call succeeds then all subsequent calls to that smart
+ * contract SHALL fail.
```

Comment:
In order to be EVM equivalent, it was determined that calls to a deleted contract shall be a no-op (execute the 0x0 op code) instead of failing.